import std;
using namespace std;

///TryThis
void TryThis2() {
	//TODO
}
void TryThis3() {
	//TODO
}
void TryThis4_7() {
	//TODO
}

///Drill

///Review

///Exercises

///Main
int main() try {
	return 0;
}
catch (exception& e) {
	cout << "Exception: " << e.what() << endl;
	return -1;
}
catch (...) {
	cout << "unhandled exception" << endl;
	return -2;
}

//resume 7.4.8 but all TT need to be done pg 308